bindgen/raze/Cargo.toml

```diff
@@ -1,9 +1,19 @@
-[raze]
-genmode = "Remote"
-workspace_path = "//bindgen/raze"
+[package]
+name = "fake_rules_rust_bindgen_lib"
+version = "0.0.1"
+
+[lib]
+path = "fake_rules_rust_bindgen_lib.rs"
 
 [dependencies]
 bindgen = "0.54.0"
+libloading = "=0.5.0"
```

Comment:
It was being manually loaded in https://github.com/bazelbuild/rules_rust/pull/432/commits/a56490da769a2f4daf8f972f9d86cf59ae7eef0c#diff-559b24b793c89733f91d4e6a1cdb1af7785f20e4748b4b40e59e68163016260cL33-L45

This seems like error initially since the block 
```toml
[raze.crates.libloading.'0.5.2']
additional_deps = [":global_static"]
additional_build_file = "libloading-global-static.BUILD"
```
was trying to set dependencies for `libloading.'0.5.2'` when in fact this version was never being specified anywhere. Raze would give a warning about this and then the BUILD file would setup `raze__libloading__0_5_0`.

By adding this dependency to the `Cargo.toml` file, this allows raze to correctly pull and setup the dependency.